### PR TITLE
Inputorama

### PR DESCRIPTION
**Pull request by sigvef**

### CODE REVIEW

File: src/ThresholdNode.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
   class ThresholdNode extends TextureGen.CanvasNode {
     constructor(id) {
-      super(id, 'Threshold', ['Image', 'Threshold']);
+      super(id, 'Threshold', [new TextureGen.GraphInput({name: 'Image'}),
+                              new TextureGen.GraphInput({name: 'Threshold'})]);
```
Comment (stianjensen) [inline src/ThresholdNode.js]: This should be a NumberInput, right?

File: src/MotionBlurNode.js [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
   class MotionBlurNode extends TextureGen.CanvasNode {
     constructor(id) {
-      super(id, 'MotionBlur', ['Image', 'Intensity']);
+      super(id, 'MotionBlur', [new TextureGen.GraphInput({name: 'Image'}),
+                               new TextureGen.GraphInput({name: 'Intensity'})]);
```
Comment (stianjensen) [inline src/MotionBlurNode.js]: NumberInput here?

File: src/DistortXNode.js [hunk 3]
```diff
@@ -3,7 +3,11 @@
 
   class DistortXNode extends TextureGen.CanvasNode {
     constructor(id) {
-      super(id, 'DistortX', ['Image', 'Map', 'Amount']);
+      super(id, 'DistortX', [
+          new TextureGen.GraphInput({name: 'Image'}),
+          new TextureGen.GraphInput({name: 'Map'}),
+          new TextureGen.GraphInput({name: 'Amount'})
```
Comment (stianjensen) [inline src/DistortXNode.js:9]: Should Amount be a NumberInput?

**Comment by sigvef** [comment]
ping

**Pull request merged**